src/App.vue

```diff
@@ -46,21 +43,19 @@ export default {
 </script>
 
 <style lang="scss">
+    @import "_scss/default";
+
     .container{
```

Comment:
Thanks, it's must have missed that.
Well.., we can do that, or change the logIn page maybe? Removing the full image from the background and redesign it to something else. How's your thought @aamesaa ?